lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_facts.py

```diff
@@ -76,6 +76,33 @@
             returned: always
             type: str
             sample: admin
+        boot_diagnostics:
+            description:
+                - Information about the boot diagnostics settings.
+            returned: always
+            type: complex
+            contains:
+                enabled:
+                    description:
+                        - Indicates if boot diagnostics are enabled.
+                    type: bool
+                    sample: true
+                storageUri:
```

Comment:
I think we should pythonize these names here for consistency

Reply:
@zikalino Could you give the name example, How to pythonize these name? Thanks!